.gitignore

```diff
@@ -49,3 +49,7 @@ dbt/dbt_packages/
 dbt/target/
 dbt/logs/
 dbt/.user.yml
+
+# memray profiles
```

Comment:
The `-d` parameter lets you put the profiles anywhere, but I assume people will end up with a few lying around in the repo that don't need to get tracked.